zeppelin-zengine/pom.xml

```diff
@@ -304,10 +304,6 @@
         </executions>
       </plugin>
 
-      <plugin>
```

Comment:
It took me a long time to figure out that this execution breaks the Maven build.
Until a few hours ago, I thought that this execution was just putting the dependencies under a location. What I didn't know is that this step also adds the dependency to the Maven reactor. This has led to various duplicate libraries within the Maven build, up to and including serious errors. It was hard to debug because the IDE ignores these dependencies. 

The funny or sad thing, the build step is completely unnecessary.

Reply:
Yeah, this kind of step is too hard to debug by IDE, but, unfortunately, Zeppelin has a bit lot of these types of modification. Hopefully, I would like to reduce/remove them.